rocks-lib/src/manifest/metadata.rs

```diff
@@ -25,7 +24,7 @@ impl<'de> serde::Deserialize<'de> for ManifestMetadata {
 impl ManifestMetadata {
     // TODO(vhyrro): Perhaps make these functions return a cached, in-memory version of the
     // manifest if it has already been parsed?
-    pub fn new(manifest: &String) -> Result<Self> {
+    pub fn new(manifest: &String) -> mlua::Result<Self> {
```

Comment:
note: we should not be returning internal mlua types, this should be wrapped in a newtype